mimicpy/__main__.py

```diff
@@ -112,30 +115,31 @@ def checkargs(args):
         print(f'\nError in arguments passed. Exiting..\n')
         sys.exit(0)
 
-def prepqm(qm, ndx, cpmd):
+def prepqm(qm, ndx, cpmd, icpmd, mdp):
```

Comment:
The meaning of icpmd is clear from the header, so maybe rename it to cpmd_template?

Reply:
Done